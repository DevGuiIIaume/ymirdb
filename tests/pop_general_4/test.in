set a 1
set b a a
pop b
list entries
bye
