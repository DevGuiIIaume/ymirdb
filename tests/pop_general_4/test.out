> ok

> ok

> a

> b [a]
a [1]

> bye
